common/init_llvm.cpp

```diff
@@ -32,6 +33,10 @@ InitLLVM::InitLLVM(int& argc, char**& argv)
   if (InitializeTargets) {
     InitializeTargets();
   }
+
+  // Printing to stderr should flush stdout. This is most noticeable when stderr
+  // is piped to stdout.
+  llvm::errs().tie(&llvm::outs());
```

Comment:
Looks good. Looking through the LLVM history, this was at some point intended to be the default behavior, but the change to make it so introduced a data race and got [partially reverted](https://github.com/llvm/llvm-project/commit/030897523d43e3296f69d25a71a140d9e5793c6a) (there's still a lingering comment in the header incorrectly saying that `errs()` is tied to `outs()` by default...).